src/Algebra/Graph/AdjacencyMap/Algorithm.hs

```diff
@@ -233,3 +233,106 @@ isTopSortOf xs m = go Set.empty xs
                   && go newSeen vs
       where
         newSeen = Set.insert v seen
+
+-- | Compute the /breadth-first search/ forest of a graph that corresponds to
+-- searching from each of the graph vertices in the 'Ord' @a@ order.
```

Comment:
Please document time and memory complexity.

Reply:
I've added a rough estimate on time and space complexity based on `bfsTreeAdjacencyMapUtil` function given that it is the most expensive one and is used by every other `bfs` algorithm.

Reply:
As I mention, Alga uses `n` and `m` throughout documentation to denote the number of vertices and edges. Please use the same notation.